composer.json

```diff
@@ -12,7 +12,7 @@
     ],
     "require": {
         "php": ">=5.3.3",
-        "symfony/yaml": "~2.0"
+        "symfony/yaml": "~2.0|~3.0"
```

Comment:
the reason why it works currently is because PHPUnit also contains the Yaml classes, and they are found from there when running the testsuite